diagnostic.md

```diff
@@ -73,13 +109,27 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "PATCH" "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data "{
+    \"donuts\": {
+      \"name\": \"French Cruller\",
+      \"price\": \"$0.99\"
+    }
+  }"
+
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (form) {
+  return $.ajax({
+    url: app.host + '/donuts/',
+    method: 'PATCH',
+    data: getFormFields(form),
```

Comment:
Assuming we don't have form fields, in this case we didn't, you would follow the same direction as the curl request. It would look like this:

```js
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```